gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java

```diff
@@ -47,7 +47,7 @@ public class GcpLogSink implements Sink {
 
   // TODO(DNVindhya): Make cloud logging service a configurable value
   private static final String SERVICE_TO_EXCLUDE = "google.logging.v2.LoggingServiceV2";
-  private static final String DEFAULT_LOG_NAME = "grpc";
+  private static final String DEFAULT_LOG_NAME = "microservices.googleapis.com%2Fobservability%2Fgrpc";
```

Comment:
- Has this been decided?
- Need to sync this with other langs (Go)
- Do we need to percent-encode "/" ? If it works without percent-encoding I would prefer to not do it

Reply:
- Yes, if no other objections from the community. This pull request give the visibility to the community as well.
- Yes, golang will be another PR.
- Yes, it's required to be url encoded:
![image](https://user-images.githubusercontent.com/20803483/166847276-0db91ed9-16b1-411d-a779-0251194acb8c.png)